internal/controller/api/v1alpha1/image_types.go

```diff
@@ -27,16 +27,17 @@ type ImageSpec struct {
 	ClusterRef   *corev1.ObjectReference `json:"clusterRef"`
 	SSHPublicKey string                  `json:"sshPublicKey,omitempty"`
 	// The name of the secret containing the pull secret
-	PullSecretRef         *corev1.SecretReference `json:"pullSecretRef"`
-	IgnitionOverrides     string                  `json:"ignitionOverrides,omitempty"`
-	StaticIpConfiguration string                  `json:"staticIpConfiguration,omitempty"`
+	PullSecretRef     *corev1.SecretReference `json:"pullSecretRef"`
+	IgnitionOverrides string                  `json:"ignitionOverrides,omitempty"`
+	StaticIpsConfig   string                  `json:"staticIpConfiguration,omitempty"`
```

Comment:
We talked about it in a different PR but it should be a struct right?

Reply:
I changed it in my image controller pr to be a struct

Reply:
> @nmagnezi is this PR still relevant? i think that #865 is doing the same no?

@filanov yup. I missed that. closing this PR.